requirements/base.txt

```diff
@@ -15,3 +15,4 @@ requests==2.23.0
 serverlessrepo==0.1.10
 aws_lambda_builders==1.1.0
 tomlkit==0.7.0
+watchdog==0.10.3
```

Comment:
How does watchdog affect what we need installed for customers? Looking at their github repo, it says you need xcode installed for Mac: https://github.com/gorakhargosh/watchdog#dependencies.

There is no `.whl` for this package in pypi as well. How does this dependency impact MSI generation and possible the pyinstaller work that is ongoing for a new way to install linux? We have had issues with libraries written in C before with our installers.